src/docs/sections/hero-banner.hbs

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  {{> html-head title="Hero Banner" }}
+</head>
+
+<body>
+  <section class="hero-carousel">
+    <div id="hero-banner-carousel" class="carousel slide" data-ride="carousel">
+      <ol class="carousel-indicators text-right d-block">
+        <li data-target="#hero-banner-carousel" data-slide-to="0" class="rounded-circle active d-inline-block"></li>
+        <li data-target="#hero-banner-carousel" data-slide-to="1" class="rounded-circle d-inline-block"></li>
+      </ol>
+      <div class="carousel-inner">
+        <div class="carousel-item active">
+          <a href="#">
+            <picture>
+              <source srcset="https://via.placeholder.com/1280x489.jpg/2596be" media="(min-width: 992px)"/>
+              <source srcset="https://via.placeholder.com/375x419.jpg/EFADBA"/>
+              <img class="w-100" alt="/" src="https://via.placeholder.com/375x419.jpg/EFADBA">
+            </picture>
+          </a>
+          <div class="carousel-caption d-none pt-2 pb-2 pr-1 pl-1 text-left">
```

Comment:
we are not going to have text for the hero banners. Text is included in the hero-banner image always